cmd/internal/cli/actions_linux.go

```diff
@@ -133,7 +133,7 @@ func execStarter(cobraCmd *cobra.Command, image string, args []string, name stri
 
 	engineConfig := singularityConfig.NewConfig()
 
-	configurationFile := files.GetSysConfigFile()
+	configurationFile := syfs.GetSysConfigFile()
```

Comment:
Can't you use `buildcfg.SINGULARITY_CONF_FILE` here?

Reply:
@sashayakovtseva FYI, that was part of the initial work: the code was full of buildcfg.SINGULARITY_CONF_FILE = "string" and at one spot it was not correctly used at all. So a function seems more useful.

Reply:
I see there is still `buildcfg. NVIDIALIBS_FILE`, so mixing func and consts seems strange.

Reply:
Good point and i do not know what was @mem intent atm

Reply:
> Can't you use `buildcfg.SINGULARITY_CONF_FILE` here?

I can, yes.

As noted in one of the commits in this PR: "Keep the GetSysConfigFile function just to make the code look a little cleaner." It is unusual for Go code to have identifiers which are all caps.